quest_test/saved-state/workflow_metadata.json

```diff
@@ -0,0 +1 @@
+{"3f9276cc-286f-436b-8435-a9c898b58137": "NestedEventFlow"}
```

Comment:
Please purge `saved-state` from the git commit. The test should probably delete the folder when it's done. 

Pytest comes with one: https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmp-path-fixture

Reply:
Done